packages/elements/src/overlay-menu/index.ts

```diff
@@ -433,8 +434,17 @@ export class OverlayMenu extends Overlay {
     });
 
     this.setItemHighlight();
+    this.restoreNestedProperties();
 
-    if (OpenedMenusManager.isNested(this)) {
+    super.onClosed();
```

Comment:
`super.onClosed()` at Overlay did nothing. Do we need this line? And is this line duplicated to line 457 in `restoreNestedProperties` function?